staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/types.go

```diff
@@ -74,6 +74,9 @@ func (item *kindItem) VisitMap(schema *openapi.Map) {
 	item.err = errors.New("expected kind, but got map")
 }
 
+func (item *kindItem) VisitArbitrary(schema *openapi.Arbitrary) {
```

Comment:
@mengqiy is that OK?

Reply:
This functions is NOP.
But IIUC we will not go deeper when we found a filed is `RawEntension`. IOW, it is not expected to reach this function. If so, it would be better to return an error.

Reply:
I've now changed this so it returns an error.

Reply:
I'm a little bit doubtful about that. But I have no knowledge about that so I'll try to reason in different terms.
- If you're right, nothing happens,
- if you're wrong, we break something :-/

I think we should leave it empty, and add an error only once we have a test to get more confidence that this is not going to blow up somewhere.

Reply:
IMO we should return the error. If it is unexpected to reach this function, we should surface the error instead of doing nothing. It may help the user to discover an error in the schema.

 > add an error only once we have a test to get more confidence that this is not going to blow up somewhere.

I'm not worrying about it much since it's at the very early point of 1.10 cycle. We have more than 3 months to discover the potential issue if any.